src/Build/Evaluation/Context/EvaluationContext.cs

```diff
@@ -57,7 +66,29 @@ internal EvaluationContext(SharingPolicy policy)
         /// </summary>
         public static EvaluationContext Create(SharingPolicy policy)
         {
-            var context = new EvaluationContext(policy);
+            
+            // ReSharper disable once IntroduceOptionalParameters.Global
```

Comment:
I don't understand this comment. Disable ReSharper once IntroduceOptionalParameters.Global...something?

Reply:
It disables a Resharper rule that wants to collapse the two Create methods into a single one with a null default for the file system. But that would break binary compat so I thought it would be safer to just have it disabled so other people don't run it at some later point in time.

Reply:
I wanted to do something similar for a different method at one point. Someone had to explain to me why I couldn't. I was sad.

Reply:
Such is the way of seeing sharp.